src/pages/Product/components/AlertModal/AlertModal.scss

```
@@ -0,0 +1,51 @@
+.AlertModal {
+  display: flex;
+
+  position: fixed;
+
+  z-index: 999;
+  top: 30px;
+  right: 100px;
+
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+
+  width: 400px;
+
+  height: 200px;
+  padding: 30px;
+  transform: translateX(0px);
+  transition: transform 200ms 1s;
+  transition-property: transform, visibility;
+  border-radius: 10px;
+  background-color: #232323;
+  box-shadow: 1px 1px 1px rgba(0, 0, 0, 0.4);
+
+  color: white;
+
+  .AlertModal-btn {
```

```suggestion
  &-btn {
```

중복되는 부분은 줄일 수 있지 않을까요?
이곳 말고 여러 곳에서도요!

---

넵 확인해서 수정하도록 하겠습니다!